types/algoliasearch/lite/index.d.ts

```diff
@@ -10,7 +10,15 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
-type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>
+import {
+  QueryParameters,
+  ClientOptions,
+  MultiResponse,
+  SearchForFacetValues,
+  Response,
+  BrowseParameters,
+  BrowseResponse
+} from '../index';
```

Comment:
imported these now! much better definition for lite (no chance we forget a duplicate)